decision_making_policy.md

```diff
@@ -0,0 +1,191 @@
+
+### Bug Report
+A [Bug Report] is used to inform the community of an error or problem encountered in the project. It allows the community to discuss the problem, and find potential solutions. 
+When the community finds agreement on how to fix the bug, you can [create a Pull Request], and the bug can be fixed from there. 
+It is customary to directly offer your fix for the problem as a pull request for review to facilitate and speed up the solution of the problem. Even if not perfect, this can also advance the discussion by discussing it more specifically.
+However, note that ultimately, [the Trusted Committer] has the mandate to reject a suggested/agreed upon course of action, if it is deemed not to serve the best for the project. 
+
+#### _Timeframe_
+Similar to feature requests, there is not a timeframe on bug reports. If fixing an error is urgent to you, make sure to put some time and effort into defining the error thoroughly and into finding a solution with the rest of the community.
+You can also offer your own version of a fix in a pull request for the bug to facilitate and speed up the solution process. 
+If [the Trusted Committer] doesn't receive any responses from the Contributor in an extended period of time, they have the mandate to close the issue containing the bug report. 
+
+## Pull requests 
+Everyone in the community can [create a Pull Request], or add reviews, comments, and suggestions to existing [Pull Requests]. 
+The process for _merging_ pull requests follows the [InnerSource approach]https://github.com/project-origin/origin-collaboration/blob/main/docs/introductory/innersource-short-role-descriptions.md#what-is-the-working-process-using-the-three-roles), where the decision of which and when pull requests get merged belongs to the [Trusted Committer]. 
+
+#### _Timeframe_
+To ensure quick merge turnarounds, the merging process for pull requests generally follows a [lazy consensus] approach, which means that community members not reacting to a pull request give their silent consent. 
+However, to ensure code quality and consistency with the goals of Project-Origin, **a pull request can only be merged when it is approved by a [Trusted Committer]**. To make sure you fulfill any merging criteria, make sure to consult the [Contribution Guidelines] before creating the pull request. 
+
+### Creating a Meeting Minutes
+There are no specific decision processes connected to submitting a meeting minutes pull request. Any decision-making processes should be specified by the submitter of the minutes pull request, or by the meeting participants if relevant. However, it is good practice as a facilitator to submit the meeting minutes pull request in good time, to allow meeting participants to add any inputs or comments before the upcoming meeting. 
```

Comment:
I would suggest this happening in a discussion instead, then physical meetings would be documented in the same manner as virtual discussions/meetings.